### DISCUSSION TITLE
Alpine-Nextcloud LXC Default username and password for app

### DISCUSSION THREAD

**Discussion by BadrEddineMoufid** [tteck/Proxmox#2318] [Help Needed] [answered]
Hello, I can't seem to find the default password for Nextcloud app after install using the Alpine-Nextcloud LXC 
and how can i connect to MySQL server installed on the LXC

**Comment by tteck** [comment]
Did you read the instructions?

**Reply by tteck** [reply, marked as answer]
> To Update Alpine Nextcloud or **Show Alpine Nextcloud Login Credentials**, run the command below in the LXC Console.

**Reply by BadrEddineMoufid** [reply]
sorry but where

**Reply by tteck** [reply]
![Screenshot 2024-01-10 11 49 59 AM](https://github.com/tteck/Proxmox/assets/21342670/cb664d43-2c89-4e8e-bc78-b8c791407a3c)

**Reply by BadrEddineMoufid** [reply]
sorry, but it does not give me the option to do so

EDIT: i'm running proxmox  7.4-13

**Reply by BadrEddineMoufid** [reply]
Thank got it to work just weird stuff when pressing the copy button
One more question how can I access the MySQL shell inside the LXC I get a:
`
alpine-nextcloud:~# mysql
-ash: mysql: not found
`

**Reply by enter360** [reply]
When I try this it does not work. It gives back the error of invalid credentials.

```
Welcome to Alpine Linux 3.19
Kernel 6.5.13-1-pve on an x86_64 (/dev/tty1)

alpine-nextcloud login: bash -c "$(wget -qO - https://github.com/tteck/Proxmox/raw/main/ct/alpine-nextcloud.sh)"
Password: 
Login incorrect
alpine-nextcloud login: 
```

**Reply by Mellowlynx** [reply]
Login first, then execute the line.
If you used the default settings when making the LXC, just entering root as user should be enough.

**Comment by keda82** [comment]
i get the option 2 to Show Alpine Nextcloud Credentials but it errors out with the following error code:

Alpine
cat: can't open 'nextcloud.creds': No such file or directory

[ERROR] in line 90: exit code 0: while executing command cat nextcloud.creds

**Reply by tteck** [reply]
The script didn't complete successfully.

**Reply by keda82** [reply]
agreed, is there any debugging I can enable?

i get the Nextcloud login prompt on https://1.2.3.4/ so the installation appears to have been made successfully

**Reply by keda82** [reply]
I reinstalled a fresh LXC and now I get the credentials

**Comment by twist3dimages** [comment]
> ![Screenshot 2024-01-10 11 49 59 AM](https://private-user-images.githubusercontent.com/21342670/295642241-cb664d43-2c89-4e8e-bc78-b8c791407a3c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1MzMxMjYsIm5iZiI6MTcxNTUzMjgyNiwicGF0aCI6Ii8yMTM0MjY3MC8yOTU2NDIyNDEtY2I2NjRkNDMtMmM4OS00ZThlLWJjNzgtYjhjNzkxNDA3YTNjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEyVDE2NTM0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZjZTBlMWUxNTc0ZGZmZjEzZjJiNWM0ZjUwNTBmYjZhM2U0ZGI2YjljODY1NzNlZjY5YjQ1ZWQ0ZWEzODlhMTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.wnbiK3aXDMwqa7Ekz4Eq3oLuk1JEyNviVRgU-Fjyo2E)


I'm confused, how do i run that in the nextcloud LXC if it's asking for a password?
![image](https://github.com/tteck/Proxmox/assets/33134502/03ecfa23-ff5c-4a2a-9fa1-c4f7866333f2)

**Reply by tteck** [reply]
❗ It is crucial not to exit the Proxmox shell while running scripts. Leaving the shell during script execution will result in the script being interrupted.

**Comment by silv3r90** [comment]
I have the Same issue, I left the proxmox shell open while the script was running.
When i try to enter the copied string into the lxc shell it's asking for a password.

![image](https://github.com/tteck/Proxmox/assets/50369316/ce1f3b2d-610f-4106-846b-b09401d273a7)

**Reply by Jey-Cee** [reply]
I think there is a huge gap between the description from tteck and the reality for the users. 
I tried also to use alpine nextcloud and ended up with the same situation. 
The installation seems to be successfully, but there is no information for login credentials. Not for nextcloud and not for the LXC. 

If I have understood tteck correctly, the console of the LXC should be opened after the installation and you should be logged in. 
But this does not seem to happen (see screenshot).

![Bildschirmfoto vom 2024-06-17 12-03-58](https://github.com/tteck/Proxmox/assets/16007760/a4a1931d-f8de-423f-92af-19bf80545a6e)

Some information:
I used the web browser to execute the install script. My browser is Firefox 127.0.

**Reply by Mellowlynx** [reply]
After the installation has completed successfully, you can open the LXC console through the Proxmox UI of ssh (`pct enter [LXC ID])
No login is needed when using the default setting. Then you can execute the command from within the LXC.

**Reply by Jey-Cee** [reply]
> After the installation has completed successfully, you can open the LXC console through the Proxmox UI of ssh (`pct enter [LXC ID]) No login is needed when using the default setting. Then you can execute the command from within the LXC.

Thanks for your reply. 
But that is not the case. As you can see i used default settings. After installation is finished, i opened the console for LXC and saw login prompt.

**Reply by tteck** [reply]
Alpine containers you have to type `root` at `Login:`

**Comment by twist3dimages** [comment]
just FYI I would add that to the documentation, it's not very clear

**Comment by Nico1788** [comment]
In the console of the alpine nextcloud container you must first type : root
then enter the command
bash -c "$(wget -qO - https://github.com/tteck/Proxmox/raw/main/ct/alpine-nextcloud.sh)"
a popup will open where Nextcloud Login Credential is selected, simply click ok and it will show you all the credentials
![proxmox alpine nextcloud](https://github.com/tteck/Proxmox/assets/174012593/ca7d4992-7496-4726-b24f-9f3df3d731a9)
